doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -44,6 +44,9 @@ With the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option enabled, the follo
 * :kconfig:option:`CONFIG_BT_FAST_PAIR_REQ_PAIRING` - The option enforces the requirement for Bluetooth pairing and bonding during the `Fast Pair Procedure`_.
   This option is enabled by default.
   See the :ref:`ug_bt_fast_pair_gatt_service_no_ble_pairing` for more details.
+* :kconfig:option:`CONFIG_BT_FAST_PAIR_SUBSEQUENT_PAIRING` - The option adds support for the Fast Pair Subsequent Pairing feature.
+  This option is enabled by default unless the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN` is enabled.
```

Comment:
```suggestion
  It is enabled by default unless the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN` Kconfig option is enabled.
```

Reply:
we have it phrased this way for 2 other entries (that were part of the NCS 2.7.0 release) so I need to align them as well

Reply:
You're right - sorry for the additional edits, just felt the text doesn't flow nicely due to repetitions